supplementary_style_guide/glossary_terms_conventions/general_conventions/n.adoc

```diff
@@ -81,17 +81,28 @@
 
 *See also*:
 
-// Azure: General; kept as is
 [discrete]
 [[dotnet]]
-==== image:images/yes.png[yes] .NET Core (noun)
-*Description*: Microsoft's newest development platform, ".NET Core", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile backends. All .NET Core applications can run on .NET Core or on the full .NET Framework. Always refer to it as .NET Core; all other variants are incorrect.
+==== image:images/caution.png[with caution] .NET (noun)
+*Description*: Microsoft's development platform, ".NET", is an open source and cross-platform framework for building cloud-based internet-connected applications, such as web apps, IoT apps, and mobile back ends. All .NET applications can run on .NET or on the .NET Framework. For version 5 and later, refer to it as ".NET". For earlier versions, use ".NET Core".
 
-*Use it*: yes
+*Use it*: with caution
 
-*Incorrect forms*: dotNet Core, .Net Core, .NET, .Net
+*Incorrect forms*: .Net, dotNet, .NET Core (incorrect for version >= 5), dotNet Core, .Net Core
```

Comment:
I don't think we want to have the caveat information `(incorrect for version >= 5)` here in the incorrect forms. You already specify that in the description above, so I don't think it's necessary to also include here. Can you remove that part please?

Reply:
removed